src/graph/optimizer/rule/LimitPushDownRule.cpp

```diff
@@ -50,6 +50,10 @@ StatusOr<OptRule::TransformResult> LimitPushDownRule::transform(
   const auto proj = static_cast<const Project *>(projGroupNode->node());
   const auto gn = static_cast<const GetNeighbors *>(gnGroupNode->node());
 
+  DCHECK(graph::ExpressionUtils::isEvaluableExpr(limit->countExpr()));
```

Comment:
Don't use DCHECK to debug your code if it's the regular branch you need to handle.

Reply:
This won't happen in correct state.

Reply:
So why did you do the same check in line 54?

Reply:
Handle exception scenes